packages/html-reporter/src/testCaseView.tsx

```diff
@@ -45,9 +47,18 @@ export const TestCaseView: React.FC<{
     return matchTags(test.title).sort((a, b) => a.localeCompare(b));
   }, [test]);
 
+  const handleCopy = (value: string) => {
+    navigator.clipboard.writeText(value).then(() => {
+      setCopyIcon(icons.check());
+      setTimeout(() => setCopyIcon(icons.copy()), 3000);
```

Comment:
You probably want to clear this timeout from an effect?

Reply:
Fixed